cmd/soroban-rpc/internal/methods/get_events.go

```diff
@@ -375,13 +402,19 @@ func (h eventsRPCHandler) getEvents(request GetEventsRequest) (GetEventsResponse
 		}
 		results = append(results, info)
 	}
+
 	return GetEventsResponse{
-		LatestLedger: latestLedger,
+		LatestLedger: ledgerRange.LastLedger.Sequence,
```

Comment:
Would be a good idea to include oldestLedger like other `getTransactions` endpoint

Reply:
Yes It is a good idea, it is out of scope for this PR, created issue here: https://github.com/stellar/soroban-rpc/issues/231